README.md

```diff
@@ -122,19 +126,25 @@ _You may also use a symlink as an alternative to copying: `ln -s sample.env .env
 4. **Start the Services**
 
 ```bash
-./manage.sh -n <network> -a start
+./manage.sh -n <network> -a up
```

Comment:
technically, this is still correct - but using the sysv standard `start` is more accurate

Reply:
Yes, technically they both work and in my opinion `start` and `stop` is more intuitive than `up` and `down`. 
However, we need to be coherent with one term or another. I find it very confusing for a user that reads `start` on the docs and on the command line you see `up`.
```bash
$ ./manage.sh 

Usage:                    
    ./manage.sh -n <network> -a <action> <optional args> 
        -n|--network: [ mainnet | testnet | mocknet | bns ] 
        -a|--action: [ up | down | logs | reset | upgrade | import | export | bns ] 
    optional args:        
        -f|--flags: [ proxy,bitcoin ] 
        export: combined with 'logs' action, exports logs to a text file 
    ex: ./manage.sh -n mainnet -a up -f proxy,bitcoin 
    ex: ./manage.sh --network mainnet --action up --flags proxy 
    ex: ./manage.sh -n mainnet -a logs export 
```

I would recommend changing it to one coherent term (any term). We change everything (including the command line output) to `start` and `stop`.?